src/components/NewsCard.tsx

```diff
@@ -3,29 +3,42 @@
 import React from 'react'
 import Markdown from 'react-markdown'
 import styled from '@emotion/styled'
-import { FontVariant, Color } from '@/app/theme'
+import { FontVariant, Color, ScreenSize } from '@/app/theme'
 import Post from '@/data/posts'
 
-// TODO: Responsive when the screen size is smaller than 1088px
-
 export const categoryColors: { [key: string]: string } = {
   publication: Color.orange900,
   award: Color.orange700,
   position: Color.orange600,
+  news: Color.orange500,
 }
 
 const postWidth = 400
 const padding = 80
 
-const PostContainer = styled.div`
+const PostContainer = styled.div<{ position: 'left' | 'right' }>`
   display: flex;
   flex-direction: column;
   width: ${postWidth}px;
   gap: 4px;
-  margin-bottom: -${padding / 4}px;
+  margin-bottom: -${padding / 2}px;
   border-radius: 8px;
   border: 1px solid ${Color.gray300};
   padding: 16px;
+  margin-${props => (props.position === 'left' ? 'right' : 'left')}: ${padding}px;
```

Comment:
There could be different implementations too.

```suggestion
  margin: auto
  ${({position}) => position === 'left'
  ? css`
      margin-right: ${padding}px;
  ` : css`
      margin-left: ${padding}px;
  `}
```

or have different styled components for left and right so that we can get rid of the prop `position`.
or we can use `nth-child()` if we always alternate between left and right. (https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)
or define, in the parent component, another styled components that decide the position of the Post only.

Reply:
I personally like the last option. What do you think?